EngineeringUnits/BaseUnits/Dimensionless/Dimensionless.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using EngineeringUnits.Units;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using Fractions;
+using EngineeringUnits.Units;
+
+namespace EngineeringUnits
+{
+
+    public partial class Dimensionless : BaseUnit
+    {
+
+        public Dimensionless()
+        {
+            Unit = DimensionlessUnit.Dimensionless.Unit.Copy();
+        }
+
+        public Dimensionless(decimal value, DimensionlessUnit selectedUnit) : base(value, selectedUnit.Unit) { }
+        public Dimensionless(double value, DimensionlessUnit selectedUnit) : base(value, selectedUnit.Unit) { }
+        public Dimensionless(int value, DimensionlessUnit selectedUnit) : base(value, selectedUnit.Unit) { }
```

Comment:
Does the second parameter need to exist? It might be desirable to force construction of Dimensionless to always use the basic unit 1 = 1 (no Mega, Kilo, etc.)